src/sensor/sensor.h

```diff
@@ -113,6 +121,9 @@ class Sensor : public QObject
     bool _connected;
     ProtocolDetector* _detector;
     QThread _detectorThread;
+    // For now this will be structures by: name, baudrate, path
```

Comment:
you've got three identifiers here, but how does this fit into the key, value?

Reply:
Then comment was wrong, it was written during the development.